apis/kafka/v1alpha1/zz_generated.resolvers.go

```diff
@@ -69,24 +68,6 @@ func (mg *Cluster) ResolveReferences(ctx context.Context, c client.Reader) error
 		mg.Spec.ForProvider.CustomClusterParameters.CustomBrokerNodeGroupInfo.SecurityGroups = reference.ToPtrValues(mrsp.ResolvedValues)
 		mg.Spec.ForProvider.CustomClusterParameters.CustomBrokerNodeGroupInfo.SecurityGroupRefs = mrsp.ResolvedReferences
 
-	}
-	if mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo != nil {
-		rsp, err = r.Resolve(ctx, reference.ResolutionRequest{
-			CurrentValue: reference.FromPtrValue(mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo.ARN),
-			Extract:      reference.ExternalName(),
-			Reference:    mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo.ARNRef,
-			Selector:     mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo.ARNSelector,
-			To: reference.To{
-				List:    &ConfigurationList{},
-				Managed: &Configuration{},
-			},
-		})
-		if err != nil {
-			return errors.Wrap(err, "mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo.ARN")
-		}
-		mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo.ARN = reference.ToPtrValue(rsp.ResolvedValue)
```

Comment:
This line is not checking than an actual arn is being resolved and instead sets the name of a resource instead of the required arn.

This would likely do the right behavior, but I see the ARN being replaced in hooks so not sure why it's even here.
```
if arn.IsARN(rsp.ResolvedValue) {
  mg.Spec.ForProvider.CustomClusterParameters.CustomConfigurationInfo.ARN = reference.ToPtrValue(rsp.ResolvedValue)
}}
```